public/v4/apps/azuracast.yml

```diff
@@ -0,0 +1,206 @@
+            - '8130:8130'
+            - '8135:8135'
+            - '8136:8136'
+            - '8140:8140'
+            - '8145:8145'
+            - '8146:8146'
+            - '8150:8150'
+            - '8155:8155'
+            - '8156:8156'
+            - '8160:8160'
+            - '8165:8165'
+            - '8166:8166'
+            - '8170:8170'
+            - '8175:8175'
+            - '8176:8176'
+            - '8180:8180'
+            - '8185:8185'
+            - '8186:8186'
+            - '8190:8190'
+            - '8195:8195'
+            - '8196:8196'
+            - '8200:8200'
+            - '8205:8205'
+            - '8206:8206'
+            - '8210:8210'
+            - '8215:8215'
+            - '8216:8216'
+            - '8220:8220'
+            - '8225:8225'
+            - '8226:8226'
+            - '8230:8230'
+            - '8235:8235'
+            - '8236:8236'
+            - '8240:8240'
+            - '8245:8245'
+            - '8246:8246'
+            - '8250:8250'
+            - '8255:8255'
+            - '8256:8256'
+            - '8260:8260'
+            - '8265:8265'
+            - '8266:8266'
+            - '8270:8270'
+            - '8275:8275'
+            - '8276:8276'
+            - '8280:8280'
+            - '8285:8285'
+            - '8286:8286'
+            - '8290:8290'
+            - '8295:8295'
+            - '8296:8296'
+            - '8300:8300'
+            - '8305:8305'
+            - '8306:8306'
+            - '8310:8310'
+            - '8315:8315'
+            - '8316:8316'
+            - '8320:8320'
+            - '8325:8325'
+            - '8326:8326'
+            - '8330:8330'
+            - '8335:8335'
+            - '8336:8336'
+            - '8340:8340'
+            - '8345:8345'
+            - '8346:8346'
+            - '8350:8350'
+            - '8355:8355'
+            - '8356:8356'
+            - '8360:8360'
+            - '8365:8365'
+            - '8366:8366'
+            - '8370:8370'
+            - '8375:8375'
+            - '8376:8376'
+            - '8380:8380'
+            - '8385:8385'
+            - '8386:8386'
+            - '8390:8390'
+            - '8395:8395'
+            - '8396:8396'
+            - '8400:8400'
+            - '8405:8405'
+            - '8406:8406'
+            - '8410:8410'
+            - '8415:8415'
+            - '8416:8416'
+            - '8420:8420'
+            - '8425:8425'
+            - '8426:8426'
+            - '8430:8430'
+            - '8435:8435'
+            - '8436:8436'
+            - '8440:8440'
+            - '8445:8445'
+            - '8446:8446'
+            - '8450:8450'
+            - '8455:8455'
+            - '8456:8456'
+            - '8460:8460'
+            - '8465:8465'
+            - '8466:8466'
+            - '8470:8470'
+            - '8475:8475'
+            - '8476:8476'
+            - '8480:8480'
+            - '8485:8485'
+            - '8486:8486'
+            - '8490:8490'
+            - '8495:8495'
+            - '8496:8496'
```

Comment:
Are these ports all required? If so, I suggest adding a warning note in `instructions.start` for the user to make sure all these ports are open and they are not used by any other applications. If they are already in-use the container fail to start up.